internal/pkg/service/buffer/sink/tablesink/storage/token.go

```diff
@@ -0,0 +1,17 @@
+package storage
+
+import (
+	"github.com/keboola/go-client/pkg/keboola"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/definition/key"
+)
+
+// Token is a Storage API Token used to create staging files and import files to the target table.
+// The token is scoped only to the target table, so there is a separate token for each definition.TableSink.
+type Token struct {
+	key.SinkKey
+	Token keboola.Token `json:"token"`
+}
```

Comment:
We need to store Storage API token, it is used to create staging file resource and to import files to the target table.